iis-wf/iis-wf-citationmatching-direct/src/main/java/eu/dnetlib/iis/wf/citationmatching/direct/service/CitationMatchingDirectReporter.java

```diff
@@ -0,0 +1,64 @@
+package eu.dnetlib.iis.wf.citationmatching.direct.service;
+
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+
+import com.google.common.collect.Lists;
+
+import eu.dnetlib.iis.common.citations.schemas.Citation;
+import eu.dnetlib.iis.common.schemas.ReportParam;
+import pl.edu.icm.sparkutils.avro.SparkAvroSaver;
+
+/**
+ * Reporter of citation matching direct job counters.<br/>
+ * It calculates citation matching direct counters and saves them
+ * as {@link ReportParam} datastore.
+ * 
+ * @author madryk
+ */
+public class CitationMatchingDirectReporter {
```

Comment:
I renamed it to `CitationMatchingDirectCounterReporter`.
Not a big fan of option to shorten this to `CounterReporter`. It would be easy to mistake this with class in fuzzy citation matching module or find that there is the only one general `CounterReporter` that could be used for every module that want to report something (usually very little attention is paid to reading import headers in java).